BrightID/src/components/Tasks/BituVerificationScreen.tsx

```diff
@@ -27,15 +28,22 @@ export const BituVerificationScreen = function ({ route }) {
     user: { verifications },
   } = store.getState();
   const connections = useSelector(selectAllConnections);
-  const bituVerification = verifications.find(
-    (v) => v.name === 'Bitu',
-  ) as BituVerification;
+  const bituVerification = (verifications.find((v) => v.name === 'Bitu') || {
+    directReports: {},
+    reportedConnections: {},
+    score: 0,
+    linksNum: 0,
+    tempScore: 0,
+  }) as BituVerification;
   const directReports = bituVerification
     ? Object.keys(bituVerification.directReports)
     : [];
   const reportedConnections = bituVerification
     ? Object.keys(bituVerification.reportedConnections)
     : [];
+  const negativeScore = bituVerification.linksNum - bituVerification.score;
+  const currentLinksNum = bituVerification.tempScore + negativeScore;
+  const releaseTime = moment(bituVerification.releaseTime).fromNow();
```

Comment:
If the user is not verified for bitu we are creating an "empty" verification object. This will result in wrong value displayed for releaseTime.